core/src/main/java/de/danoeh/antennapod/core/service/playback/PlaybackService.java

```diff
@@ -887,7 +887,6 @@ public void playerError(PlayerErrorEvent event) {
         if (mediaPlayer.getPlayerStatus() == PlayerStatus.PLAYING) {
             mediaPlayer.pause(true, false);
         }
-        PlaybackPreferences.writeNoMediaPlaying();
```

Comment:
This line was added in d2084608a67bdd9359e8ec1dcea92acc30f577e6 as a bug fix. Is that bug fix no longer relevant? Or will removing the line here make that  bug reappear?

Reply:
Thank you @ByteHamster for the review,
The indicated commit [d208460](https://github.com/AntennaPod/AntennaPod/commit/d2084608a67bdd9359e8ec1dcea92acc30f577e6) does not point to any changes on the changed line. The changes were introduced by the commit 0eeb47ee1e2ad4d2f91c1851480af80fcc364181. From the testing done there are no bugs introduced by our changes.

If you need any further help, please feel free to revert back.
Thank you again.!

Reply:
0eeb47ee1e2ad4d2f91c1851480af80fcc364181 just moved the line from another place. Originally, it was introduced in d2084608a67bdd9359e8ec1dcea92acc30f577e6 to fix a bug.

Reply:
After extensive testing the reappearing notification bug does not reappear after this change. We also tested against this https://github.com/AntennaPod/AntennaPod/pull/3350 and did not see any regressions.

Reply:
The problem with this is that the reappearing notification bug only happened for very few users, if I remember correctly. I don't think I was ever able to reproduce this locally -- but the change fixed it for these users. This makes it quite hard to test whether the issue was re-introduced.

Reply:
Thank you @ByteHamster once again for the review. 
We understand your point and we will look for another way to solve this issue.